codegen/tests/simple/servicehandler.go

```diff
@@ -80,6 +81,38 @@ func (s *ServiceHandler) GetApiDocsListHandler(w http.ResponseWriter, r *http.Re
 	restlib.SendHTTPResponse(w, httpstatus, apidoc)
 }
 
+// GetGetSomeBytesListHandler ...
+func (s *ServiceHandler) GetGetSomeBytesListHandler(w http.ResponseWriter, r *http.Request) {
+	if s.serviceInterface.GetGetSomeBytesList == nil {
+		common.HandleError(r.Context(), w, common.InternalError, "not implemented", nil, s.genCallback.MapError)
+		return
+	}
+
+	ctx := common.RequestHeaderToContext(r.Context(), r.Header)
+	ctx = common.RespHeaderAndStatusToContext(ctx, make(http.Header), http.StatusOK)
+	var req GetGetSomeBytesListRequest
+
+	ctx, cancel := s.genCallback.DownstreamTimeoutContext(ctx)
+	defer cancel()
+	valErr := validator.Validate(&req)
+	if valErr != nil {
+		common.HandleError(ctx, w, common.BadRequestError, "Invalid request", valErr, s.genCallback.MapError)
+		return
+	}
+
+	client := GetGetSomeBytesListClient{}
+
+	pdf, err := s.serviceInterface.GetGetSomeBytesList(ctx, &req, client)
+	if err != nil {
+		common.HandleError(ctx, w, common.DownstreamUnexpectedResponseError, "Downstream failure", err, s.genCallback.MapError)
+		return
+	}
+
+	headermap, httpstatus := common.RespHeaderAndStatusFromContext(ctx)
+	restlib.SetHeaders(w, headermap)
+	restlib.SendHTTPResponse(w, httpstatus, pdf)
```

Comment:
@gkanz the `pdf` type is `type Pdf []byte` (and of course a pointer at this point). It is passed directly to `restlib.SendHTTPResponse` which attempts to cast it to a `[]byte` which fails. we need this code to pass the correct type down please

Reply:
I have updated the `svc_handler.sysl` and sent you the diff.